roles/dnsmasq/tasks/resolvconf.yml

```diff
@@ -23,7 +37,9 @@
 - name: generate search domains to resolvconf
   set_fact:
     searchentries:
-      "{{ ([ 'default.svc.' + dns_domain, 'svc.' + dns_domain ] + searchdomains|default([])) | join(' ') }}"
+      search {{ ([ 'default.svc.' + dns_domain, 'svc.' + dns_domain ] + searchdomains|default([])) | join(' ') }}
```

Comment:
this is a repeat of roles/dnsmasq/tasks/dnsmasq.yml L12-15. Can this be a preinstall set_fact?